source/nodejs/ac-typed-schema/src/markdown/generate-markdown.ts

```diff
@@ -1,15 +1,46 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { I18n } from "i18n";
 import { defined } from "./defined";
 import { defaultValue } from "./defaultValue";
 import { SchemaClass } from "../SchemaClass";
 import { SchemaEnum } from "../SchemaEnum";
 import { SchemaLiteral } from "../SchemaLiteral";
 import { SchemaProperty } from "../SchemaProperty";
 import { SchemaPropertyType } from "../SchemaPropertyType";
+import * as path from "path";
+import * as console from "console";
 import * as mdTable from "markdown-table";
 import * as style from "./style";
 
+class MarkdownConfig {
+    i18n : I18n;
+
+    constructor() {
+        this.i18n = new I18n();
+        this.i18n.configure({
+            locales: ['en', 'de', 'sp'],
+            directory: path.join(__dirname, '../../src/markdown/languages'),
+            syncFiles: true
+        });
+    }
+
+    _locale : string = "en";
```

Comment:
Should this be declared up top with the other variable? Also, can it be initialized by the URL or default browser language?

Reply:
should be private as well -- I'll fix it up :)